kaitai_utils.py

```diff
@@ -2,13 +2,22 @@
 import importlib
 import inspect
 from io import BufferedWriter
-import json
-from json_stream import streamable_dict, streamable_list
 import os
 from typing import Any, BinaryIO, Dict, Generator, List, Type
-import yaml
 
+import hansken_extraction_plugin
+from hansken_extraction_plugin.api.extraction_trace import ExtractionTrace
+import json
+from json_stream import streamable_dict, streamable_list
 from kaitaistruct import KaitaiStruct
+import yaml
+
+
+def write_kaitai_to_trace(trace: ExtractionTrace):
```

Comment:
since this is a public method, can you add a pydoc to this class?

Reply:
done